ToDoListAPI/ToDoListAPI/DTOs/TaskRequest.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ToDoListAPI.Models
+{
+    public class TaskRequest
+    {
+        [Required]
+        public int Id { get; set; }
+        [Required]
+        public string Name { get; set; }
```

Comment:
```suggestion
        public string Name { get; set; }
```
Name cx cần validate length nhé